comms/shared/peering/sps.go

```diff
@@ -84,9 +84,10 @@ var (
 
 	stagingEndpoint = `https://api.thegraph.com/subgraphs/name/audius-infra/audius-network-goerli`
 
+	// TODO: (remove) select foundation nodes only as peers for development and testing
```

Comment:
this looks fine to me temporarily, but is the EnableJetstream variable not enough to effectively do the same thing?